app/src/main/java/ca/ualberta/CMPUT3012019T02/alexandria/activity/SignUpActivity.java

```diff
@@ -0,0 +1,46 @@
+package ca.ualberta.CMPUT3012019T02.alexandria.activity;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+
+import ca.ualberta.CMPUT3012019T02.alexandria.R;
+
+public class SignUpActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_sign_up);
+    }
+
+    public void addPhoto(View view) {
+        //TODO Implement
+        throw new UnsupportedOperationException("Not implemented");
+    }
+
+    public void signUp(View view) {
+        //TODO Implement
```

Comment:
maybe mention here that a call will be being made to verify that the username and email of the user are unique

Reply:
Updated with comments